### PR TITLE
New config option 'opstimeout' has been added.

### PR DESCRIPTION
**Pull request by plageat**
This is regarding issue #14

### CODE REVIEW

File: mock/py/mockbuild/util.py [hunk 1]
```diff
@@ -620,6 +622,10 @@ def do(command, shell=False, chrootPath=None, cwd=None, timeout=0, raiseExc=True
 
     return output
 
+def do(command, *args, **kargs):
+    timeout = kargs.get('timeout', _OPS_TIMEOUT)
+    f = functools.partial(_do, timeout=timeout)
+    return f(command, **kargs)
```
Comment (xsuchy) [inline mock/py/mockbuild/util.py]: Why so complicated. Cannot be just 
```
def do(... timeout=None, ..):
if timeout is None:
   timeout = _OPS_TIMEOUT
```

**Comment by xsuchy** [comment]
Beside those inline comments. I like this patch. Once you address those inline comments I will merge it.

File: mock/py/mockbuild/util.py (hunk 1 above)
Comment (plageat) [reply mock/py/mockbuild/util.py]: Agree.I will re-work this

File: mock/py/mockbuild/util.py [hunk 2]
```diff
@@ -620,6 +621,10 @@ def do(command, shell=False, chrootPath=None, cwd=None, timeout=0, raiseExc=True
 
     return output
 
+def do(command, *_, **kargs):
+    if 'timeout' not in kargs:
+        kargs['timeout'] = _OPS_TIMEOUT
```
[the remark this answers is not in the document]
Comment (plageat) [reply mock/py/mockbuild/util.py]: Agree, there is no reason. Will be re-worked

**Comment by xsuchy** [comment]
Thank you.

**Pull request merged**